source/extensions/common/wasm/v8/v8.cc

```diff
@@ -144,15 +161,6 @@ class V8 : public WasmVm {
   bool module_needs_emscripten_{};
 };
 
-template <typename T> struct V8ProxyForGlobal : Global<T> {
```

Comment:
What's up with all the code-shuffling? I don't mind that you moved code to a more fitting place, but it's a lot of unnecessary changes in already huge PR, and it makes it impossible to spot the differences in shuffled code...

Could you please move those functions back to the old places, and we can move them again in a separate PR?

Reply:
Didn't realized I had move them... they are back PTAL.